fhir-persistence-jdbc/src/test/java/com/ibm/fhir/persistence/jdbc/test/util/ParameterExtractionTest.java

```diff
@@ -89,7 +472,6 @@ public void testRange() throws FHIRPersistenceProcessorException {
     private String timestampToString(java.sql.Timestamp tstamp) {
         // do not use Timestamp#toString() because it converts to local timezone.
         // We need it rendered in UTC
-        return tstamp.toInstant().atZone(ZoneOffset.UTC).format(DATE_TIME_FORMATTER);
+        return tstamp.toInstant().atZone(ZoneOffset.UTC).format(TIMESTAMP_FORMATTER);
     }
-
 }
```

Comment:
optionally, even though his may have been covered by those new persistence layer tests. but seems still better to add a e2e test in server integration tests to cover the touchstone medicationknoledge search case mentioned in issue #348.

Reply:
I think the unit tests are sufficient in this case.  My goal is to be very comprehensive in the unit tests, but i don't think we need 100% coverage for the e2e suite.